diff --git a/node_modules/@clack/prompts/dist/index.mjs b/node_modules/@clack/prompts/dist/index.mjs
index abb011d..0978feb 100644
--- a/node_modules/@clack/prompts/dist/index.mjs
+++ b/node_modules/@clack/prompts/dist/index.mjs
@@ -571,7 +571,7 @@ ${styleText("gray", S_BAR)}` : ""}` : ""}`;
             options: this.options,
             cursor: this.cursor,
             maxItems: u.maxItems,
-            columnPadding: t.length,
+            columnPadding: t.replace(/\u001b\[[0-9;]*m/g, "").length,
             rowPadding: o + n,
             style: c
           }).join(`
@@ -587,7 +587,7 @@ ${s}
             options: this.options,
             cursor: this.cursor,
             maxItems: u.maxItems,
-            columnPadding: t.length,
+            columnPadding: t.replace(/\u001b\[[0-9;]*m/g, "").length,
             rowPadding: s + o,
             style: c
           }).join(`
@@ -793,7 +793,7 @@ ${styleText("gray", S_BAR)}` : ""}`;
             options: this.options,
             cursor: this.cursor,
             maxItems: n.maxItems,
-            columnPadding: i.length,
+            columnPadding: i.replace(/\u001b\[[0-9;]*m/g, "").length,
             rowPadding: s + v,
             style: g
           }).join(`
@@ -809,7 +809,7 @@ ${u}
             options: this.options,
             cursor: this.cursor,
             maxItems: n.maxItems,
-            columnPadding: i.length,
+            columnPadding: i.replace(/\u001b\[[0-9;]*m/g, "").length,
             rowPadding: u + s,
             style: g
           }).join(`
@@ -1120,7 +1120,7 @@ ${styleText("gray", S_BAR)}` : ""}`;
             cursor: this.cursor,
             options: this.options,
             maxItems: e.maxItems,
-            columnPadding: r.length,
+            columnPadding: r.replace(/\u001b\[[0-9;]*m/g, "").length,
             rowPadding: g + h,
             style: (p, b) => a(p, p.disabled ? "disabled" : b ? "active" : "inactive")
           }).join(`
